### PR TITLE
Update URL for SEO

### PR DESCRIPTION
**Pull request by saracope**
**Summary**

This PR fixes/implements the following **bugs/features**

* Modify URL's to include '!' in the new format of /#!/explore-code
* Update hardcoded URL's

Explain the **motivation** for making this change. What existing problem does the pull request solve?
* SEO improvements

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

**Code formatting**

**Closing issues**

Fixes #

### CODE REVIEW

**Comment by saracope** [comment]
Update is in dev @jlow81.

Will generate new sitemap after production update.

**Comment by saracope** [comment]
[SEO Improvements Phase 1 #Sprint_32](https://trello.com/c/oriTRlPQ/806-seo-improvements-phase-1-sprint32)

**Comment by DanielJDufour** [comment]
@saracope , could you update https://github.com/GSA/code-gov-web/blob/master/src/app/utils/urls.ts#L16?  `toRouterLink` is what converts the urls in the configuration JSON file to Angular router links, which allow us to switch pages with reloading. Thanks!

Also, open to suggestions, renaming of functions, or better ways to doing things.  It was what I came up with, but there might be a better practice.  :-)

**Comment by saracope** [comment]
Will do, thanks @DanielJDufour.

**Comment by DanielJDufour** [comment]
@saracope , I think there's some merge conflicts because of some of the url fixes in the code-dot-gov.json.  Can you resolve the conflicts? And then we can merge.

File: config/examples/code-dot-gov.json [hunk 1]
```diff
@@ -99,7 +99,11 @@
           "description": "A python package designed to simulate and analyze resilience of water distribution networks.",
           "image": "/assets/img/featured-projects/pipes4.png",
           "links": [
+<<<<<<< HEAD
```
Comment (DanielJDufour) [inline config/examples/code-dot-gov.json]: There's a merge conflict here.

**Comment by DanielJDufour** [comment]
@saracope , I'm ready to merge this as soon as the merge conflicts are resolved :-)

**Comment by DanielJDufour** [comment]
@saracope , I just merged the changes.  Please delete the `sc_seo-update` branch if you think that you won't use it again.  Thanks!

**Pull request merged**